x-pack/plugins/security_solution/public/common/links/structure.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { SecurityPageName } from '../../../common/constants';
+import { EXPLORE, INVESTIGATE } from '../../app/translations';
+import { FEATURE, LinkItem } from './types';
+import { links as hostsLinks } from '../../hosts/links';
+import { links as detectionLinks } from '../../detections/links';
+import { links as networkLinks } from '../../network/links';
+import { links as usersLinks } from '../../users/links';
+import { links as timelinesLinks } from '../../timelines/links';
+import { getCasesLinkItems } from '../../cases/links';
+import { links as managementLinks } from '../../management/links';
+import { links as overviewLinks, landingLinks, detectionResponseLinks } from '../../overview/links';
+
+export const appLinks: LinkItem[] = [
+  overviewLinks,
+  landingLinks,
+  detectionResponseLinks,
+  detectionLinks,
+  {
+    id: SecurityPageName.explore,
+    title: EXPLORE,
+    path: 'to do',
```

Comment:
Shoud this be?
```suggestion
    id: SecurityPageName.threatHunting,
    title: THREAT_HUNTING,
    path: '/threat_hunting',
```

Reply:
yes i believe these should be your paths but i was leaving them to you to fill in the blanks since they're not put into use in this PR